app/models/request.rb

```diff
@@ -27,4 +28,32 @@ def items_hash
       [key, Item.order("created_at ASC").find_by(partner_key: key)]
     end.to_h
   end
+
+  def family_request_reply
+    {
+      "organization_id": organization_id,
+      "partner_id": partner_id,
+      "requested_items": item_requests.map do |ir|
+        {
+          "item_id": ir.item_id,
+          "count": ir.quantity
```

Comment:
could you include `ir.item.name`? The Partner side won't have the name info...I think?